### ISSUE TITLE
Runtime training DeepYeast

### ISSUE THREAD

**Issue by melanibe** [dimkou/DL-AtlasKaggle#8]
I ran the DeepYeast on GPU (files have been modified for GPU run in my mel branch - not send a pull request for now). So i checked that the system really used GPU but ... after 24hours not a single epochs was done... so i added save training metrics every 10 steps and val set also to see what is happedning and actually after 24hours only 10 steps were done.... the problem is the validation step (i.e. predict on the whole validation set which was set to 20% of the training set i.e. rgouhly 5000 images) took 10,000s.... and otherwise EACH single training step took 1000s with batch_size 32. these are figures on GPU run. On my computer with batch size 32 it is not even managing to do one single training step.
